artemis-commons/src/main/java/org/apache/activemq/artemis/utils/UTF8Util.java

```diff
@@ -34,7 +35,7 @@
 
    private static final boolean isTrace = ActiveMQUtilLogger.LOGGER.isTraceEnabled();
 
-   private static final ThreadLocal<SoftReference<StringUtilBuffer>> currenBuffer = new ThreadLocal<>();
+   private static final FastThreadLocal<SoftReference<StringUtilBuffer>> currentBuffer = new FastThreadLocal<>();
```

Comment:
This only benefits if all threads are FastThreadLocalThread's. Do we re-use Netty's thread pools? I thought we had our own....

Reply:
@michaelandrepearce +1

Reply:
@michaelandrepearce Good catch! I will change it in no time, thanks!